src/calculateRentalCost.js

```diff
@@ -28,7 +28,28 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = 0;
+  const basicCost = 40;
+  const smallerOff = 20;
+  const biggerOff = 50;
+
+  if (days < 3) {
+    result += basicCost * days;
+
+    return result;
+  }
+
+  if (days >= 3 && days < 7) {
```

Comment:
You can get rid of using && if you start checking from the highest condition to the lowest.
You should also look at the checkpoint 3